src/api/types.ts

```diff
@@ -62,7 +62,7 @@ export interface HideProps {
 export interface OHRIFormSchema {
   name: string;
   pages: Array<OHRIFormPage>;
-  processor: string;
+  processor: FormProcessor;
```

Comment:
Doesn't this introduce breaking changes to already existing forms?

Reply:
this is noted let me fix this